extensions/MacOSX/kDriveLiteSync/Extension/main.m

```diff
@@ -16,8 +16,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#import "../../../../src/libcommonserver/io/fileAttributes.h"
```

Comment:
Move the common declarations in vfs.h ?
Or create a new dynamic library "libcommonserverext" with the common code.